pages/[username]/[collectionId]/[nftId]/index.tsx

```diff
@@ -3,33 +3,56 @@ import { GetServerSideProps } from 'next';
 import GalleryRedirect from 'scenes/_Router/GalleryRedirect';
 import { MetaTagProps } from 'pages/_app';
 import { openGraphMetaTags } from 'utils/openGraphMetaTags';
-import { graphql, useLazyLoadQuery } from 'react-relay';
-import { NftIdQuery } from '__generated__/NftIdQuery.graphql';
-import GalleryRoute from 'scenes/_Router/GalleryRoute';
+import { DecoratedCloseIcon } from 'src/icons/CloseIcon';
+import styled from 'styled-components';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import useKeyDown from 'hooks/useKeyDown';
+import { useCallback } from 'react';
 
 type NftDetailPageProps = MetaTagProps & {
   nftId: string;
   collectionId: string;
 };
 
 export default function NftDetailPage({ collectionId, nftId }: NftDetailPageProps) {
-  const query = useLazyLoadQuery<NftIdQuery>(
```

Comment:
this query was moved one level lower to the child component, since that'll be re-used with the modal